### ISSUE TITLE
[1.12.2] Suggestion - Invisible Light Config Option

### ISSUE THREAD

**Issue by MokahTGS** [romelo333/notenoughwands#77]
I was wondering if there could be a config option added to the Illumination wand that changed the light balls to an invisible version that still lights everything up but doesn't have the balls showing.  They could show up as balls if you are holding the wand, but disappear when you're not.

There is a mod called Lightair that adds light balls that do just this, but I like the wands that use RF.

https://minecraft.curseforge.com/projects/lightair

**Comment by Matakor** [comment]
Alternatively, could we maybe have small light balls? Like, 1/4th the size of the current ones?

**Comment by Guswut** [comment]
If possible could we get it set up so the mode switch key toggles the illumination wand as suggested by the two other people?

Specifically, this would be what I'd think would be an ideal list of modes:

**Normal full sized light**: Identical to the current light.
**Dim full sized light**: A reduced alpha version (maybe 20%?) that is dimmer than the current light but produces the same amount of light.
**Ethereal full sized light**: Like the current light but it can only be removed by right clicking it with the wand, so no hitbox to get in the way.
**Ethereal dim full sized light**: A reduced alpha version that is dimmer than the current light but produces the same amount of light and it can only be removed by right clicking it with the wand, so no hitbox to get in the way.
**Normal small light**: A smaller textured version (maybe 1/4th?) of the current light
**Dim small light**: A smaller textured version of a reduced alpha version of the small light.
**Ethereal small sized light**: A smaller textured version of the current light but it can only be removed by right clicking it with the wand, so no hitbox to get in the way.
**Ethereal dim full sized light**: A smaller textured version of the current light and reduced alpha version that is dimmer than the current light but produces the same amount of light and it can only be removed by right clicking it with the wand, so no hitbox to get in the way.
**Ethereal ghostly light**: An invisible textured version which can only be seen when holding the wand in this mode, and can only be removed by right-clicking the light with the wand (or placing a block over it).

Thank you!

**Comment by 00xZEROx00** [comment]
Yeah that's be cool, because for me who use shaders, and sphax texture packs, the orbs are rendered incorrectly showing up a black box who follow the player camera...
I have searched in the config, and there is no way to hide the orb texture and so on...

**Comment by 00xZEROx00** [comment]
Ok solved... just make a blank transparent sprite in resourcepack and orbs are gone, now there is only light...

Structure:
```
assets
|_notenoughwands
    |_textures
        |_blocks
            |_light.png
            |_light.png.mcmeta
```

**light.png**
You can use GIMP, or Aseprite
Sprite Size: 64x384
Transparent!

**light.png.mcmeta**
```json
{
  "animation": {
    "frametime": 3
  }
}

```
